deliaioanac/assignment1/CaseSensitive.cpp

```diff
@@ -0,0 +1,80 @@
+#include <stdio.h>
+#include <string>
+#include <vector>
+#include <algorithm>
+
+using namespace std;
```

Comment:
Oh, I see that I guessed correctly when replying to the above comment on using namespace haha. I'll change it right away, it will give the code more readability.